prismic-model/js/manual-promos.js

```diff
@@ -0,0 +1,15 @@
+import structuredText from './parts/structured-text';
+import image from './parts/image';
+import link from './parts/link';
+
+const ManualPromo = {
+  Contact: {
+    title: structuredText('Title', 'single', ['heading1']),
+    format: link('Format', 'document', ['article-formats', 'event-formats']),
+    summary: structuredText('Summary', 'single'),
```

Comment:
We don't use `summary` anywhere. Given this stuff is so hard to change, I think we should go for consistency over correctness here, we have `description` or `caption` all over the place.